docs/semgrep-appsec-platform/jira.md

```diff
@@ -49,10 +51,11 @@ To enable the Jira integration, follow these steps:
 Once you have enabled the Jira integration, you must complete the following steps on Semgrep AppSec Platform's **Integrations** page:
 
 1. Select the **Subdomain** for the Jira instance you want to use.
-2. Select the **Default project** where the Jira tickets will be created.
-3. Select the **Issue type** you want created with your Semgrep findings.
-4. Optional: To see an example of the content Semgrep populates your Jira ticket with, click **See preview**.
-5. Click **Save changes** to proceed.
+1. Select the **Default project** where the Jira tickets will be created.
+1. Select the **Issue type** you want created with your Semgrep findings.
+1. Optional: To see an example of the content Semgrep populates your Jira ticket with, click **See preview**.
+1. Optional: Click **Customize ticket creation**, then select **<i class="fa-solid fa-square-check"></i> Automatically create tickets for high severity, high confidence findings on the default branch** to allow Semgrep to automatically create tickets.
```

Comment:
@s-santillan Would it be worth calling out that you can configure automations separately for each product and that each product has a different definition for what gets created? E.g. supply chain includes reachability?

Reply:
![image](https://github.com/semgrep/semgrep-docs/assets/20766840/164d08a0-dc11-4421-8430-08eb825780cc)